website/profile/views.py

```diff
@@ -353,9 +354,10 @@ def user_account_password(auth, **kwargs):
         user.change_password(old_password, new_password, confirm_password)
         user.save()
     except ChangePasswordError as error:
-        push_status_message('<br />'.join(error.messages) + '.', kind='warning')
+        for m in error.messages:
+            push_status_message(m, kind='warning', trust=False)
```

Comment:
I'm preeeetty sure this was something we had to fix at some point for a similar endpoint, but not necessarily in this view. Ideally, of course, the front-end should be validating and not sending the request until it's pretty much ready, which it may be and this might not really be a problem. 

The least impactful would be to join with `'. '` instead. Hmmm. Let's keep it the way you have it and, if it's terrible, we'll fix it properly later.